oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/authentication/OAuth2DeviceAuthorizationRequestAuthenticationToken.java

```diff
@@ -96,7 +96,7 @@ public OAuth2DeviceAuthorizationRequestAuthenticationToken(Authentication client
 		this.deviceCode = deviceCode;
 		this.userCode = userCode;
 		this.authorizationUri = null;
-		this.additionalParameters = null;
+		this.additionalParameters = Map.of();
```

Comment:
Please use `Collections.emptyMap()` to be consistent with other usages in the framework - `Map.of()` is not currently used.

Reply:
Done!